### PR TITLE
feat(pages: main): 增加隐藏(归档)todo的功能

### PR DESCRIPTION
**Pull request by tirelyl**

### CODE REVIEW

File: src/app/pages/main/right-control/todo/todo.component.html [hunk 1]
```diff
@@ -1,47 +1,72 @@
...
+          <li nz-menu-item *ngIf="currentContextTodo?.listUUID !== 'todo'" (click)="moveToList('todo')">
+            默认列表
+          </li>
+          <li nz-menu-item *ngFor="let list of listsExcept(currentContextTodo.listUUID)" (click)="moveToList(list._id)">
+            {{ list.title }}
+          </li>
+        </ul>
+      </li>
+    </ng-container>
     <li nz-menu-divider></li>
```
Comment (wzhudev) [inline src/app/pages/main/right-control/todo/todo.component.html:65]: This divider shouldn't be there if the `Todo` is archived.

File: src/app/pages/main/right-control/todo/todo.component.ts [hunk 2]
```diff
@@ -68,8 +70,11 @@ export class TodoComponent implements OnInit, OnDestroy {
       })
       .map(todo => Object.assign({}, todo) as Todo)
       .sort(rankerGenerator(rank));
-
-    this.todos = [].concat(filteredTodos);
+    this.todos = [ ...filteredTodos ];
+    this.all = this.todos.filter(todo => !todo.archived);
+    this.archived = this.todos.filter(todo => todo.archived);
+    console.log(this.todos);
```
Comment (wzhudev) [inline src/app/pages/main/right-control/todo/todo.component.ts:76]: Please remove console.

**Review by wzhudev — CHANGES_REQUESTED** [review]
Your code is clean to read. Changes requested are about user experience.

Toggling completed tasks is not likely to be a setting that users want to change all the time. Instead of a `tab`, modification on the sort menu would be preferred. Like, create a new menu in replacement of sort menu. Make sort menu as a sub menu, and toggling completed tasks as a menu item.

And please provide a online demo.

**Comment by tirelyl** [comment]
换种交互实现一下，close it

**Pull request closed without merging**